examples/interactive-polygon/app/build.gradle

```diff
@@ -23,13 +23,6 @@ android {
         versionCode 1
         versionName "1.0"
 
-        if (keystoreProperties.containsKey("MAPLIBRE_API_KEY")) {
-            resValue "string", "maplibre_api_key", keystoreProperties["MAPLIBRE_API_KEY"]
-        } else {
-            println("WARNING: MAPLIBRE_API_KEY is missing from keystore file!")
-            resValue "string", "maplibre_api_key", ""
-        }
-
```

Comment:
Ahh, okay... I had not considered that, but I think it makes sense.

I have just pushed some changes that do the following:

1. Uses the `MAPLIBRE_STYLE_URL` property instead
2. Sets the resource `maplibre_style_url` instead of the API key, defaulting to the demo tiles
3. Loads that resource for this demo

This should allow a balance of not requiring an API key for basic testing, but allowing the use of any desired vector style for the interactive polygon demo.